lib/ui/src/components/sidebar/Tree/Tree.tsx

```diff
@@ -106,7 +119,9 @@ const Tree: FunctionComponent<{
       return <List>{children.map(mapNode)}</List>;
     }
     case node.isLeaf: {
-      return <Leaf key={node.id} {...node} depth={depth} isSelected={selected[node.id]} />;
+      return (
+        <Leaf key={node.id} tags={tags} {...node} depth={depth} isSelected={selected[node.id]} />
```

Comment:
There should be no need to pass in `tags` explicitly here, should be part of `node`